packages/shared/src/components/sidebar/Sidebar.spec.tsx

```diff
@@ -135,32 +127,30 @@ it('should not render create my feed button if user has alerts.filter as false',
 });
 
 it('should remove filter alert if the user has filters and opened feed filters', async () => {
-  let mutationCalled = false;
   mockGraphQL({
     request: {
       query: UPDATE_ALERTS,
       variables: { data: { filter: false } },
     },
     result: () => {
-      mutationCalled = true;
       return { data: { _: true } };
     },
   });
   renderComponent({ filter: true });
 
-  await act(async () => {
-    const feedFilters = await screen.findByText('Feed filters');
-    feedFilters.click();
-  });
-
-  await waitFor(() => {
-    const key = getFeedSettingsQueryKey(defaultUser);
-    const data = client.getQueryData(key) as AllTagCategoriesData;
-    expect(getHasAnyFilter(data.feedSettings)).toBeTruthy();
-  });
-
-  expect(updateAlerts).toBeCalledWith({ filter: false });
-  expect(mutationCalled).toBeTruthy();
+  // await act(async () => {
```

Comment:
Just wanted to check if this was expected to be commented out?

Reply:
Oh good point, I think this was actually meant to be refactored as `act` is not optimal.
Let me double-check this one 🤯